packages/components/src/components/link/link.stories.ts

```diff
@@ -32,297 +37,131 @@ export const Default = {
-                surroundingContent ? 'Qui do.' : ''
-              }%TEMPLATE%${surroundingContent ? 'Eiusmod minim excepteur.</div>' : ''}`
-            }
-          ],
-          args,
-          options: {
-            title: surroundingContent ? 'example with surrounding content' : 'example without surrounding content'
-          }
-        })
-      )}
-    `;
-  }
+export const Standalone = {
+  name: 'Standalone',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/" standalone>
+        <sd-icon library="global-resources" name="system/picture" slot="icon-left"></sd-icon>
+        Standalone
+      </sd-link>
+
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/">
+        <sd-icon library="global-resources" name="system/picture" slot="icon-left"></sd-icon>
+        Without Standalone
+      </sd-link>
+    </div>
+  `
 };
 
 /**
- * Use the part selector to align the icon within the component.
+ * Remove the `href` to disable the link.
  */
 
-export const IconAlignment = {
-  parameters: {
-    controls: { exclude: ['icon-right', 'default', 'standalone', 'base'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-link::part(base){align-items: ...;}',
-          values: ['start', 'center', 'end'].map(alignIcon => {
-            return {
-              value: `<style>#align-icon-${alignIcon} sd-link::part(base){align-items: ${alignIcon};}</style><div id="align-icon-${alignIcon}">%TEMPLATE%</div>`,
-              title: alignIcon
-            };
-          })
-        }
-      },
-      constants: [
-        {
-          type: 'template',
-          name: 'constant-template',
-          value: '<style>div[id^=align-icon-]{text-align: left; width: 300px;}</style>'
-        },
-        {
-          type: 'slot',
-          name: 'icon-right',
-          value: '<sd-icon library="global-resources" name="system/arrow-right" slot="icon-right"></sd-icon>'
-        },
-        {
-          type: 'slot',
-          name: 'default',
-          value: 'In dolore consectetur do excepteur tempor occaecat magna anim esse sit dolor mollit est voluptate.'
-        },
-        {
-          type: 'attribute',
-          name: 'standalone',
-          value: true
-        }
-      ],
-      args
-    });
-  }
+export const Disabled = {
+  name: 'Disabled',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/">Enabled</sd-link>
+      <sd-link>Disabled</sd-link>
+    </div>
+  `
 };
 
 /**
- * Use the `base`, `label`, `icon-left` and `icon-right` part selectors to customize the button.
+ * Use the `icon-left` and `icon-right` slots to add icons to each side of the link. They automatically adapt the size.
```

Comment:
* Use the `icon-left` and `icon-right` slots to add system icons to each side of the link. They automatically adapt the size.

Reply:
Updated